src/main/perl/ComponentProxy.pm

```diff
@@ -367,6 +367,33 @@ sub _setDependencies
 }
 
 
+=item _version_check
+
+Apply version related checks.
+
+Return C<SUCCESS> if there are no version-related issues;
+report an error and return undef otherwsie.
```

Comment:
I see "version-related issues" is deliberately kept undefined, I guess pending resolution of the discussion in #41? So the doc is accurate, but potentially a little confusing ?